config/templates/logging/configmap.yaml

```diff
@@ -5,5 +5,5 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: kubearchive-logging
-  namespace: {{ .Release.Namespace }}
+  namespace: kubearchive
```

Comment:
Should we have here the `app.kubernetes.io/part-of: kubearchive `label?

Reply:
Added